spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,25 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+  context "#get_cis_all" do
```

Comment:
Generally speaking we tend to prefer that, yes - however, the other example groups in this file use `context` and adhering to the rest of the file is usually a good thing, so this is also acceptable IMO. Newline would be good though :)